src/sass/mixins/_General.Mixins.scss

```diff
@@ -152,7 +152,7 @@
 
 // Ensures the block expands to the full height to enclose its floated childen.
 @mixin ms-clearfix {
-  *zoom: 1;
+  // *zoom: 1;
```

Comment:
Same with this—is there a reason for removing this?

Reply:
Was causing build issues but per our conversation about the asterisk being an IE7 hack i've removed the asterisk but will potentially remove the rule entirely as I don't believe we have floats in core.